### DISCUSSION TITLE
Cannot filter Kanban view on custom fields

### DISCUSSION THREAD

**Discussion by xavierraffin** [community/community#7301] [Projects and Issues] [answered]
I have custom a field named "category"

I am able to display it on the kanban

But when I filter (either by clicking on the category etiquette in an issue or by using the filtering bar) by it there are no results (where I should have many). 

Example filtering URL:
https://github.com/orgs/xxx/projects/3/views/3?filterQuery=-category%3A%22%E2%8F%B1++job+performance+%26+alerts%22

**Comment by xavierraffin** [comment, marked as answer]
I found the issue: there was a whitespace in front of the field name `" Category"` instead of `"Category"`

Still not ideal that it breaks the filtering

**Reply by azenMatt** [reply]
Glad you got this sorted, we will take a closer look at how we handle the spaces 👍
